### DISCUSSION TITLE
📝Comments of [kdt/2024/04/11/sql-init]

### DISCUSSION THREAD

**Discussion by giscus** [PresentKim/legacy-blog-sveltekit#19] [Announcements]
# 📝Comments of [kdt/2024/04/11/sql-init]

오늘은 오라클 SQL을 설치하고 SQL에 대해 공부했습니다.

https://present.kim/posts/kdt/2024/04/11/sql-init

**Comment by PresentKim** [comment]
asklfsdflkgjdsflksd
